docs/development/room-dag-concepts.md

```diff
@@ -38,16 +38,14 @@ Most-recent-in-time events in the DAG which are not referenced by any other even
 The forward extremities of a room are used as the `prev_events` when the next event is sent.
 
 
-## Backwards extremity
+## Backward extremity
 
 The current marker of where we have backfilled up to and will generally be the
 oldest-in-time events we know of in the DAG.
```

Comment:
... or rather, the `prev_events` of the oldest-in-time events we know of.

Reply:
> ... or rather, the prev_events of the oldest-in-time events we know of.

They are the oldest-in-time events we know of but sometimes we only know their `event_ids`. Suggestion to phrase it better?